app/decorators/user_decorator.rb

```diff
@@ -2,7 +2,7 @@
 
 UserDecorator = Struct.new(:user) do
   def lockout_time_remaining
-    (Devise.allowed_otp_drift_seconds - (Time.zone.now - user.second_factor_locked_at)).to_i
+    (Devise.direct_otp_valid_for - (Time.zone.now - user.second_factor_locked_at)).to_i
```

Comment:
For more flexibility, perhaps add a new separate setting for lockout period versus how long the OTP is valid? We might want to have 2 different values. For example, we might want OTPs to be valid for only 5 minutes, but we want to lock malicious users out for 10 minutes.

Alternatively, we could use the existing `Devise.unlock_in` setting because we would presumably want to lock out users for the same amount of time whether they were locked out due to failed email/password attempts or failed OTP attempts.